apps/src/lib/kits/maker/ui/setup-instructions.module.scss

```diff
@@ -0,0 +1,24 @@
+:export {
+  checklistId: connectYourBoardChecklist; 
```

Comment:
That's a good question tbh. I dunno, I guess I just was thinking they're id names so they're part of the CSS. but they aren't actually used anywhere in the CSS so maybe it makes more sense to put them in SetupInstructions.

Reply:
Unless the `id` is assigned a previously defined variable/prop, it seems like you can just use the string like [this example](https://github.com/code-dot-org/code-dot-org/blob/f357947e70c84eafd0a0420f35b5e12a29e4ffa8/apps/src/lib/kits/maker/ui/SetupGuide.jsx#L66) in the same file. You can see other examples in [this file](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/dance/ai/DanceAiModal.tsx).

Reply:
I do still want them to be defined as a variable since they are referenced in 2 different places (because the anchor tag uses the value of the id to jump there)

Reply:
Gotcha - that makes sense. Thanks!